readme.md

```diff
@@ -90,10 +90,14 @@ If you want to construct a single object with multiple state-picks inside, simil
 import { create } from 'zustand'
 import { useShallow } from 'zustand/react/shallow'
 
-const useBearStore = create((set) => ({
-  bears: 0,
-  increasePopulation: () => set((state) => ({ bears: state.bears + 1 })),
-  removeAllBears: () => set({ bears: 0 }),
+const useBearStore = create(set => ({
+  nuts: 0,
+  honey: 0,
+  treats: {},
+  increaseNuts: () => set(state => ({ nuts: state.nuts + 1 })),
+  increaseHoney: () => set(state => ({ honey: state.honey + 1 })),
+  increaseTreats: (treat: string, count?: number) =>
+    set(state => ({ treats: { ...state.treats, [treat]: (state.treats[treat] ?? 0) + 1 } })),
 }))
```

Comment:
Hi, thanks for the suggestion. I didn't realize this.
We don't want to increase lines readme.md (and possibly move detail/complete examples to docs),
so can we make this simpler??

```suggestion
const useBearStore = create((set) => ({ nuts: 0, honey: 0, treats: {} /* ... */ }))
```

Reply:
Thanks, and I agree.  The update functions are unnecessary, so I removed them and added a `...`, which I think is clear enough.  I also added a link to what documentation I could find for `createWithEqualityFn`.

I'm afraid I don't have the bandwidth to go deeper and break this into its own file, etc.  Hopefully this will be sufficient for now.  Thanks for the feedback!